README.md

```diff
@@ -8,19 +8,50 @@
 
 > An ActionCable companion for Y.js clients
 
-This is a monorepo with [npm modules](./packages) and [Ruby gems](./gems).
-Please check the respective sub-repos for more information. 
+TODO: Explain what this library does.
 
-## Usage
+## Components
+
+TODO: list sub-modules and link to their READMEs. The README should explain what
+each sub-component accomplishes, and how it cooperates with the other components.
+
+## Setup
+
+This is a monorepo built and laid out with [turborepo](https://turbo.build).
+The following dependencies need to exist in order to build this repository
+with `turbo` and run the sample application:
+
+- `Ruby`
+- `yarn`
+- `Node.js/npm`
+
+Refer to the respective configuration files to see which versions are required.
+The easiest option is to use `asdf` to install and manage these tools, but you
+may also refer to `/.tool-versions` and install these manually on your host machine.
+
+## Build
 
-Install gem and npm package:
+Run the following commands from the top-level directory of this repository.
 
+Install `turbo`:
+
+```shell
+$ npm install turbo --global
 ```
-gem install y-rb_actioncable
-yarn add @y-rb/actioncable
+
+Install JavaScript dependencies:
+
+```shell
+$ yarn
 ```
 
-### Example
+Build the repository:
+
+```shell
+$ turbo build
+```
+
+## Usage
```

Comment:
@mttkay sorry for not coordinating better. I have already added a development specific section now https://github.com/y-crdt/yrb-actioncable#development to help with onboarding specifically for contributors. There are a few differences:
- We don't need to install turbo manually, it will just be there when running `yarn` the first time. `yarn` is capable of resolving binaries from the `.bin` folder in the `node-modules` directory, so a global installation isn't necessary (and we can control the turbo version).
- I kept the usage instructions, because they are important for actual users
- Added build & deploy information (including bumping the version, changesets and information about GitHub Actions used).
- Haven't added a top-level description for all packages (for me that was covered by _Usage_), but I will take what you have in your PR and add it.

Reply:
OK, thanks this is great. Just a not that I did not remove the Usage section; the diff just pushed it below the `Setup` section.

Appreciate the help!

Reply:
Should I close this then?

Reply:
@mttkay I would just take this PR, rebase from main, fix the few conflicts and merge than. Ok?

Reply:
Yes sure, thanks :+1:

Reply:
> the diff just pushed it below the Setup section.

🤦 missed that.